Bottom: 7e0ba628a7fb27ad1ec5333d59e936bb8412f4f8
Top:    9f12c640e20d3c2bb03dececc000be8c2b96cc99
Author: scott snyder <sss@karma>
Date:   2020-02-12 11:38:17 -0500

Refresh of cmake

---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 762d809..4b4439d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -89,7 +89,7 @@ ADD_SHARED_LIBRARY( ${PROJECT_NAME} ${library_sources} )
 
 ADD_SHARED_LIBRARY( ${PROJECT_NAME}Processors ${processor_srcs} ${diagnostics_srcs} )
 
-TARGET_LINK_LIBRARIES( ${PROJECT_NAME} ${MarlinUtil_LIBRARIES} ${LCIO_LIBRARIES} )
+TARGET_LINK_LIBRARIES( ${PROJECT_NAME} ${Marlin_LIBRARIES} ${MarlinUtil_LIBRARIES} ${LCIO_LIBRARIES} )
 TARGET_LINK_LIBRARIES( ${PROJECT_NAME}Processors ${Marlin_LIBRARIES} ${AIDA_LIBRARIES} ${PROJECT_NAME} )
 
 INSTALL_SHARED_LIBRARY( ${PROJECT_NAME} DESTINATION lib )
